http-server/src/main/resources/openapi.yaml

```diff
@@ -3,92 +3,189 @@ info:
   title: XTDB
 servers:
   - url: /
+components:
+  schemas:
+    Timeout:
+      description: timeout in milliseconds
+      type: integer
+    DefaultTz:
+      type: string
+      example: "America/Los_Angeles"
+    SystemTime:
+      type: string
+      example: "2024-01-01T00:00"
```

Comment:
examples for date-time here should end in `Z`. is there a `format` for 'date-time with TZ', or is this included in `date-time`?